transformer/src/test/scala/uk/ac/wellcome/transformer/receive/SQSMessageReceiverTest.scala

```diff
@@ -1,4 +1,6 @@
-package uk.ac.wellcome.transformer.receive
+package uk.ac.wellcome.sqs
```

Comment:
Should this still be in the transformer? I see the package name has changed but the directory is still the same

Reply:
Oops, I moved this back and forth. It still needs to live in the transformer, because it has transformer-specific parts, but I’m putting some extra tests in the common lib based on these.

Good spot, thanks!